js/game.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-named-as-default-member */
+/* eslint-disable import/no-named-as-default */
+import Scene1 from './scene1.js';
+import Scene2 from './scene2.js';
+import Scene3 from './scene3.js';
+import Board from './board.js';
+import Player from './player.js';
+
+const Game = (() => {
+  const config = {
+    // eslint-disable-next-line no-undef
+    type: Phaser.AUTO,
+    width: window.innerWidth,
+    height: window.innerHeight,
+    parent: true,
+    dom: {
+      createContainer: true,
+    },
+    scene: [Scene1, Scene2, Scene3],
+  };
+
+  const status = (() => {
+    const win = () => 'win';
+    const playing = () => 'playing';
+    const draw = () => 'draw';
+
+    return { win, playing, draw };
+  })();
+
+  // lb = logic board
+  const inspectStatus = (lb) => {
```

Comment:
This function might be better in another object/module. This alone is overriding game's single responsibility (which is to keep making players play the game until one has won or a tie is reached). This `inspectStatus` as I see can be a utility function that `game` just "mixes" in.